src/lib.rs

```diff
@@ -152,15 +154,15 @@ impl fmt::Display for CellErrorType {
 }
 
 /// File types
-enum FileType {
+enum FileType<RS> where RS: Read + Seek {
```

Comment:
We can remove the `Seek` bound if `Xls` is implemented without seeking. If the API changes to split the `FileType` and the current tuple struct member of each variant, we can immediately remove the `Seek` bound for all formats other than `Xls`.

Reply:
Is this really important to remove it? We can use a `Cursor` for if needed.

Reply:
Using `Cursor` is only possible if the full file is in memory. Maybe not critical, but this bound should be reviewed before stabilizing the API.